xarray/tests/test_cupy.py

```diff
@@ -48,3 +48,11 @@ def test_check_data_stays_on_gpu(toy_weather_data):
     """Perform some operations and check the data stays on the GPU."""
     freeze = (toy_weather_data["tmin"] <= 0).groupby("time.month").mean("time")
     assert isinstance(freeze.data, cp.core.core.ndarray)
+
+
+def test_where():
+    from xarray.core.duck_array_ops import where
+
+    data = cp.zeros(10)
+
+    assert where(data < 1, 1, data).all()
```

Comment:
should we also test that `where` returns a `cupy` array?

Reply:
Done, thanks @dcherian